### ISSUE TITLE
Artisan 2.4 Comparator profiles will not load.

### ISSUE THREAD

**Issue by dalmoody** [artisan-roaster-scope/artisan#498]
Mac Pro Mini, Sierra, BC-2MD roaster.
[artisan-settings.txt](https://github.com/artisan-roaster-scope/artisan/files/5104215/artisan-settings.txt)

Upon selecting files in Comparator; graphs should be generated as described in the instruction manual.

Files to be compared can be selected.  They are given a color code and name of file is displayed under title.  Files can be added and deleted.  No comparison graph is generated.

I have posted this question on Home Barista Forums; Roasting.

Here are the files I am trying to compare.
[Costa Rica La Minita Tarrazu_20-08-03_1258.txt](https://github.com/artisan-roaster-scope/artisan/files/5104139/Costa.Rica.La.Minita.Tarrazu_20-08-03_1258.txt)
[Costa Rica La Minita Tarrazu_20-08-17_1321.txt](https://github.com/artisan-roaster-scope/artisan/files/5104140/Costa.Rica.La.Minita.Tarrazu_20-08-17_1321.txt)

## Steps to Reproduce the Problem

  1. 
  2. 
  3. 

## Specifications

  - Artisan Version:
  - Artisan Build (number in brackets shown in the about box):
  - Platform (Mac/Windows/Linux + OS version):
  - Connected devices or roasting machine:
  
Please attach your current Artisan settings file (as exported via menu Help >> Save Setings as *.aset) file.
Please attach any relevant Artisan *.alog profiles.

_Note that you need either add a `.txt` extension or zip the files before uploading. Otherwise you will receive a "Not a supported file type" error on uploading._

**Comment by roasterdave** [comment]
Hi Dal,
Can you please confirm that you have at least BT checked under the top pull down?  Your profiles work fine for me here.  The screenshot shows them in the Comparator with BT, deltaBT, and Events checked.  Thanks.
Dave

![Issue 498](https://user-images.githubusercontent.com/30009312/90807988-2df20000-e2d4-11ea-91b2-e98480acbe5d.png)

**Comment by MAKOMO** [comment]
Hm. Works for me as well. I did a factory reset on the official v2.4 macOS build, loaded your settings, started the comparator and "Add"ed your two profiles. Are you sure you sent us the Artisan settings file and the profiles you tested with?

<img width="809" alt="Screenshot 2020-08-20 at 20 54 16" src="https://user-images.githubusercontent.com/13909011/90813222-90c0b700-e327-11ea-9e4d-3ffec13e3c30.png">

**Comment by dalmoody** [comment]
It works for me as well.  I’m very sorry for the inquest/issue.  Yes, Dave I checked the proper boxes, BT delta BT, events, and burner.  Align at charge.  After making a profile selection with the add button, I expected the comparison graph to appear in place of the selection window.  Instead the graph appears behind the window.  When I minimized the selection window, the graph appeared.

I’m old, so I use Artisan with a Mac Pro Mini and a large screen.  Artisan profiles a roast for me in the full screen mode.  It’s terrific.  That’s how the comparator issue occurred for me.

Again, I am very sorry if I have caused any alarms.  Artisan is a terrific roasting software.  You have been a great help.

Dal Moody
